Update images for control panel and add net 9 preview support (#887)
telerik/maui-docs: get-started/mac/first-steps-nuget.md

While installing Telerik UI for .NET MAUI with NuGet works both for Windows and MacOS machines, this tutorial describes how to get up and running with the library by downloading and installing the controls on macOS. Here is the step by step guide you have to follow:

> >important The `Telerik.UI.for.Maui` NuGet works with .NET 7.0, .NET 8.0 and .NET 9 at least Preview 6 projects and automatically restores the required packages depending on the .NET version you are using in your project.

Reviewer: ```suggestion
>important The `Telerik.UI.for.Maui` NuGet package supports .NET 7.0, .NET 8.0, and .NET 9 Preview 6 or later projects, and automatically restores the required packages depending on the .NET version you are using in your project.
```
